wemake_python_styleguide/constants.py

```diff
@@ -281,7 +281,7 @@
 
 #: List of bad magic module functions.
 MAGIC_MODULE_NAMES_BLACKLIST: Final = frozenset((
-    '__getattr__',
+    '__getaddr__',
```

Comment:
Sorry, I don't understand what are you doing.

Maybe I can help you with some explanations?

Reply:
I read the comment of ``BadMagicModuleFunctionViolation`` and it says 

> Forbids to use ``__getaddr__`` and ``__dir__`` module magic methods. 

So I looked into the implementation and found that it checks if ``node.name`` is in ``constants.MAGIC_MODULE_NAMES_BLACKLIST``, 

But in ``MAGIC_MODULE_NAMES_BLACKLIST``, ``__getattr__`` was in the blacklist, not ``__getaddr__``. So, I thought that this was the cause of the problem.  Maybe I misunderstood the task.

It would be great if you can provide some explanations about how to solve this problem.

Reply:
Oh, sorry. `__getaddr__` is a typo. We only work with `__getattr__`. Can you please submit a fix for it?

The problem I have described in this particular issue is that we don't correctly check where `__getattr__` is located. Here: https://github.com/wemake-services/wemake-python-styleguide/blob/d01851a23366aecf4f581f0f7032ba3511d85bdd/wemake_python_styleguide/visitors/ast/modules.py#L87

We need to check that `if isinstance(get_context(node), ast.Module):` and write a new regression test here: https://github.com/wemake-services/wemake-python-styleguide/blob/d01851a23366aecf4f581f0f7032ba3511d85bdd/tests/test_visitors/test_ast/test_modules/test_magic_module_functions.py